packages/app/src/elements/audit.ts

```diff
@@ -91,46 +69,45 @@ export class Audit extends AuditMixin(StateMixin(Routing(View))) {
         `;
     }
 
-    async audit() {
-        const { vaults } = this.state;
-
-        const { reusedPasswords, weakPasswords, compromisedPasswords } = await this.auditVaults(vaults);
-
-        // This makes the UI update
-        this._reusedPasswords = reusedPasswords;
-        this._weakPasswords = weakPasswords;
-        this._compromisedPasswords = compromisedPasswords;
-
-        this._countElements.forEach((countElement) => {
-            animateElement(countElement, { animation: "bounce" });
-        });
-    }
-
-    private _renderSection(listItems: ListItem[], label: string, routeParams: any) {
+    private _renderSection(listItems: { item: VaultItem; vault: Vault }[], type: AuditResultType) {
         return html`
-            <section class="box count" ?hidden=${listItems.length === 0}>
```

Comment:
I thought it was better to always display all sections, even if there are no results in them (yet).